src/app/Scenes/MyCollection/Screens/Insights/MedianSalePriceChart.tsx

```diff
@@ -79,20 +77,22 @@ export const MedianSalePriceChart: React.FC = () => {
               <Flex
                 position="absolute"
                 justifyContent="center"
+                alignItems="center"
                 top={CHART_HEIGHT / 2}
-                left={(CHART_WIDTH - PADDING_LEFT_AND_RIGHT) / 2 - PADDING_LEFT_AND_RIGHT}
-                width={CHART_WIDTH - PADDING_LEFT_AND_RIGHT}
+                width={CHART_WIDTH}
               >
-                <Text variant="sm" color="black60">
-                  {hasInsights
-                    ? "Incomplete data for the selected medium"
-                    : "No data currently available"}
-                </Text>
-                {!hasInsights && (
+                <Flex>
                   <Text variant="sm" color="black60">
-                    Try a different timeframe or artist
+                    {hasInsights
+                      ? "Incomplete data for the selected medium"
+                      : "No data currently available"}
```

Comment:
have you considered this and also setting the textAlign to `center`. I think design wants this centered and the text to be all together
```suggestion
                      : "No data currently available Try a different timeframe or artist"}
```

Reply:
The original ticket specified that the next be on a new line.
I think it is centered properly now - on tabs and tiny devices alike.

Reply:
ah oki - I thought they want it all centered. I am hating this centering label thing 😄